mantle/kola/tests/ignition/luks.go

```diff
@@ -176,6 +192,74 @@ func runTest(c cluster.TestCluster, tpm2 bool, threshold int, killTangAfterFirst
 	ut.LUKSSanityTest(c, tangd, m, tpm2, killTangAfterFirstBoot, rootPart)
 }
 
+func runCexTest(c cluster.TestCluster) {
+	var err error
+	var m platform.Machine
+
+	// To prevent the test to fail the whole run on s390x machine that does not have Cex Device
+	cex_uuid := os.Getenv("KOLA_CEX_UUID")
+	if cex_uuid == "" {
+		c.Skip("No CEX device found in KOLA_CEX_UUID env var")
+	}
+
+	ignition := conf.Ignition(`{
+		"ignition": {
+			"version": "3.5.0"
```

Comment:
I think for now this needs to be

```suggestion
			"version": "3.5.0-experimental"
```

Once we stabilize the spec, it'll be `3.5.0`.

Reply:
Hi @jlebon ,
Shall i wait till the new ignition gets stablized?